src/Symfony/Component/Config/Resource/GlobResource.php

```diff
@@ -95,6 +99,16 @@ public function getIterator()
 
         if (0 !== strpos($this->prefix, 'phar://') && false === strpos($this->pattern, '/**/') && (\defined('GLOB_BRACE') || false === strpos($this->pattern, '{'))) {
             foreach (glob($this->prefix.$this->pattern, \defined('GLOB_BRACE') ? GLOB_BRACE : 0) as $path) {
+                if (is_dir($path)) {
+                    if ($this->forExclusion) {
+                        yield $path => new \SplFileInfo($path);
+                        continue;
+                    }
+
+                    if (array_key_exists($path, $this->excludedPrefixes)) {
```

Comment:
this requires normalizing the directory separators in the path before using it as a key, as done in FileLoader